<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSqlRelationalTableModel的使用</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>651</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="separator"/>
   <addaction name="action_add"/>
   <addaction name="action_insert"/>
   <addaction name="action_delete"/>
   <addaction name="separator"/>
   <addaction name="action_save"/>
   <addaction name="action_cancel"/>
   <addaction name="separator"/>
   <addaction name="action_blockList"/>
   <addaction name="action_quit"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_add">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_insert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_delete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_cancel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_blockList">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>字段列表</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
